filesystem/samsung_android_pit.ksy

```diff
@@ -0,0 +1,134 @@
+meta:
+  id: samsung_android_pit
+  title: Samsung Partition Information Table
+  license: CC0-1.0
+  file-extension: pit
+  endian: le
+  encoding: utf-8
+  #imports:
+  #  - /common/samsung_signature
+
+doc: |
+  Reversed format of partition information table used in
+  samsung decices.
+
+doc-ref:
+  - https://gitlab.com/BenjaminDobell/Heimdall/-/tree/master/libpit/source  # most of the fields
+  - https://www.droidviews.com/need-know-samsung-pit-files/
+  - https://github.com/atdog/pitparser/blob/master/pitparser.py  # `signature` is documented there
+  - https://www.droidviews.com/how-to-extract-pit-file-from-samsung-galaxy-devices/  # this page also contains links to the files, to download them from Google Drive you need to construct the URI https://docs.google.com/uc?export=download&id=<file id (a string in the URI looking random, but it is not)>
+  - https://source.android.com/devices/bootloader/partitions-images
+
+seq:
+  - id: magic
+    type: u4
+    valid: 0x12349876
```

Comment:
Either use `contents` with the `[0x76, 0x98, ...]` syntax, or at least visually separate the bytes:

```suggestion
    valid: 0x12_34_98_76
```

The advantage of `u4` + `valid` is that it will be slightly faster, that's right.